### PR TITLE
Adding missing overrides to core

### PR DESCRIPTION
**Pull request by loumalouomega**
Using clang-tidy

### CODE REVIEW

File: kratos/includes/chunk.h [hunk 1]
```diff
@@ -78,7 +78,7 @@ namespace Kratos
 	  }
 
       /// Destructor is not virtual. This class can not be drived.
-	  ~Chunk() noexcept {
+	  ~Chunk() noexcept override {
```
Comment (jcotela) [inline kratos/includes/chunk.h:81]: This one is completely at odds with the comment just above. If the comment is right, this should not be an override (but may be a final). @pooyan-dadvand can you confirm?

Comment (pooyan-dadvand) [reply kratos/includes/chunk.h:81]: I agree with @jcotela and it should be a final!

**Review by pooyan-dadvand — CHANGES_REQUESTED** [review]
What I don't like is that it removes the virtual in derived class (which does not change the behavior) but not adding override so in derived classes is not clear which methods are virtual...

**Comment by loumalouomega** [comment]
Do I discard this PR?

**Comment by pooyan-dadvand** [comment]
I would also put the overrides when removing the virtuals. and the final for chunk is important!

**Comment by pooyan-dadvand** [comment]
@loumalouomega I think you miss understood me...

My suggestion was to keep the changes adding the overrides

**Comment by loumalouomega** [comment]
Ok

**Comment by RiccardoRossi** [comment]
is this PR still active or should we close it and eventually generate a new one?

**Comment by loumalouomega** [comment]
We can close it

**Pull request closed without merging**